### PR TITLE
Correct Java22 support changes

### PR DESCRIPTION
**Pull request by russgold**
The changes I had made for Java22 support turn out to have been incorrect.

While there was indeed an _ensureInitialized_ method added to Lookup, it turns out to be of limited value unless you are defining a new class. At the recommendation of the JDK team, I have switched to loading the class from the classloader.

This change also restores the Primitives class, which is needed for the ORB.

### CODE REVIEW

**Review by smillidge — APPROVED** [review]
It would be better to use a logger rather than System.out.println in the error handling. I will hit Approve to enable you to proceed.

**Comment by russgold** [comment]
> It would be better to use a logger rather than System.out.println in the error handling. I will hit Approve to enable you to proceed.

Thanks; you are correct. I see that pfl does have support for logging, but I have not used it. Are you familiar with the mechanism? It looks as though it is tied to exceptions. Is it documented?

**Comment by dmatej** [comment]
> > It would be better to use a logger rather than System.out.println in the error handling. I will hit Approve to enable you to proceed.
> 
> Thanks; you are correct. I see that pfl does have support for logging, but I have not used it. Are you familiar with the mechanism? It looks as though it is tied to exceptions. Is it documented?

I believe you can use Java standards:
```
private static final Logger LOG = System.getLogger(Bridge.class.getName());
...
- and then use it for logging when appropriate. Don't log exceptions before you throw them, they will be usually logged where they are processed.

File: pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java [hunk 1]
```diff
@@ -118,9 +119,14 @@ private MethodHandles.Lookup getLookup(Class<?> anchorClass) throws IllegalAcces
...
         try {
+            /**/
+            Class.forName(cl.getName(), true, cl.getClassLoader());
+            /*/
             getLookup(cl).ensureInitialized(cl);
-        } catch (IllegalAccessException e) {
-            e.printStackTrace();
+            /**/
+        } catch (/*IllegalAccessException | */ClassNotFoundException e) {
+            System.out.println("Ignore failure to initialize class " + cl + " due to " + e);
```
Comment (dmatej) [inline pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java:128]: `LOG.log(Level.ERROR, "Ignore failure to initialize class " + cl, e);`

File: pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java [hunk 2]
```diff
@@ -106,7 +106,8 @@ public Class<?> defineClass(Class<?> anchorClass, String className, byte[] class
         try {
             return getLookup(anchorClass).defineClass(classBytes);
         } catch (IllegalAccessException e) {
-            throw new RuntimeException("Unable to define class ", e);
+            System.out.println("Unable to define class " + className + " using anchor " + anchorClass + " due to " + e);
```
Comment (dmatej) [inline pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java:109]: I would not log it here, but I would use the message in the thrown exception:
```
throw new IllegalStateException("Unable to define class " + className + " using anchor " + anchorClass, e);
```
Also I prefer ISE to RTE, but it is not so important.

File: pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java [hunk 3]
```diff
@@ -118,9 +119,14 @@ private MethodHandles.Lookup getLookup(Class<?> anchorClass) throws IllegalAcces
     @Override
     public void ensureClassInitialized(Class<?> cl)  {
         try {
+            /**/
+            Class.forName(cl.getName(), true, cl.getClassLoader());
+            /*/
```
Comment (dmatej) [inline pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java:124]: I would also remove the commented out code.

Comment (russgold) [reply pfl-basic/src/main/java17/org/glassfish/pfl/basic/reflection/Bridge.java]: > I would also remove the commented out code and empty comments.

Yes, those were part of my experiment for how to get this to work. I have now cleaned them up.

**Pull request merged**